lib/git-environment.ts

```diff
@@ -1,5 +1,11 @@
 import * as path from 'path'
 
+function resolveEmbeddedGitDir(): string {
+  const s = path.sep;
```

Comment:
It would be great to have the same semicolon policy all over the project if feasible. As I see, you are not using them... (Except [here](https://github.com/desktop/dugite/blob/master/lib/git-process.ts#L91))

Reply:
@kittaakos oops, this should be caught by tslint [[rule here]](https://github.com/desktop/dugite/blob/master/tslint.json#L22-L25) but I guess I'm not actually running it anywhere. I'll follow up after this, but I think I'll just move to [prettier](https://github.com/prettier/prettier) as that simplifies a bunch of things.